storage/StorageExample/Shared/ImagePicker.swift

```
@@ -126,7 +127,7 @@ import SwiftUI
           }
         }
       } catch {
-        print("Image was not properly loaded.")
+        print("Image was not properly loaded.\(error)")
```

`throw` this error (here and throughout) instead of returning nil.